src/Command/ListGithubSlackMappingsCommand.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Command;
+
+use App\Repository\GitHubSlackMappingRepositoryInterface;
+use Symfony\Component\Console\Attribute\AsCommand;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Helper\Table;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+#[AsCommand(
+    name: 'gsm:list',
+    description: <<<'EOF'
+Display all GitHub repository-to-Slack mappings.
+This command provides an overview of configured mappings, including repositories, Slack channels, and mentions.
+EOF
+)]
+final class ListGithubSlackMappingsCommand extends Command
+{
+    public function __construct(
+        private readonly GitHubSlackMappingRepositoryInterface $repository
+    ) {
+        parent::__construct();
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output): int
+    {
+        $mappings = $this->repository->findAll();
+
+        if ($mappings === []) {
+            $output->writeln('<comment>No GitHub-to-Slack mappings found.</comment>');
+
+            return Command::SUCCESS;
+        }
+
+        $table = new Table($output);
+        $table
```

Comment:
[Minor] You can easily reduce one line without loosing readability thanks to fluent setters
```suggestion
        $table = (new Table($output))
```